org-repo-migrator.sh

```diff
@@ -104,53 +184,62 @@ main()
-        for tag in ${image_tags}
-        do	
-          # Check whether the repo is public/private repository	    
-          if [[ "${repo_visibility}" = "${visibility}" ]]; then
-            echo "Pulling ${name} with tag ${tag} from source ${src}"
-            # Pulling repository from the source organzation    
-            docker pull ${src}/${name}:${tag} > /dev/null
-
-            echo "Pulling repository ${name}:${tag} successful" 
-            echo "Tagging the repository from ${src}/${name}:${tag} to ${dest}/${name}:${tag}"
-            # Tagging a repository with tag to to destination org with tag
-            docker tag ${src}/${name}:${tag} ${dest}/${name}:${tag} > /dev/null
-
-            echo "Repository ${name}:${tag} tagged successfully"
-            echo "Pushing to ${dest} organization the ${name}:${tag} repository"
-            # Pushing the repository to destination org with specific tag
-            docker push ${dest}/${name}:${tag} > /dev/null
-
-            echo "Push successful for ${name}:${tag}"
-          else
-            # If repo is a private repository, skip the execution   
+    # Function to fetch the repositories in every page
+    fetchRepos "${URL}" "${VERSION}" "${src}" "${repo_page}" "${size_of_page}" list_of_repos repo_names
+    # Loop over the number of repositories in source organization 
+    for i  in $repo_names
+    do
+      # Fetch the name of the repository
+      name=$(echo ${i} | sed -e 's/\"//g' -e 's/=.*//')
+      # If condition to check whether the repository is to be skipped
+      if [[ ! "${skip_repos[@]}" =~ "${name}" ]]; then
+        # Fetch the repository privacy whether public/private repository    
+        repo_visibility=$(echo $i | sed -e 's/\"//g' -e 's/.*=//g')
+        # Fetch the image tags for the repos
+        for (( tag_page=1;;tag_page++ ));
+        do
+          # Fetch the tags for the repository
+	  fetchTags "${URL}" "${VERSION}" "${src}" "${name}" "${tag_page}" "${size_of_page}" list_of_tags tags
+          # Loop to fetch a tag from source org repos and apply to the destination org repos
+	  for tag in $tags
+          do
+            # Check whether the repo is public/private repository	    
+            if [[ "${repo_visibility}" = "${visibility}" ]]; then
+              # Function to pull a repository from source organization
+              pullRepos ${src} ${name} ${tag}
+	      # Function to tag repository from source organization to destination organization
+              tagRepos ${src} ${name} ${tag} ${dest}
+	      # Function to push repository to destination organization
+              pushRepos ${dest} ${name} ${tag}	      
+	    else
+              # If repo is a private repository, skip the execution   
+              continue
+	    fi
+          done
+	  # Check if the tag_next is null or not for tags
+	  if [[ ! $list_of_tags = "null" ]]; then 
+            # If the tag_next is not null continue looping
```

Comment:
Keep const spacing for indentation. Here and at other places we have used 4 spaces
```suggestion
          # If the tag_next is not null continue looping
```